data/categories/vp_vehicles_parts.yml

```diff
@@ -4828,3 +4829,170 @@
   - upholstery_pattern
   - watercraft_features
   - yacht_type
+- id: vp-2-4
+  name: Commercial Vehicles
+  children:
+  - vp-2-4-1
+  - vp-2-4-2
+  - vp-2-4-3
+  - vp-2-4-4
```

Comment:
For consistency, I would place these categories one level below, under `Vehicles & Parts > Vehicles > Motor Vehicles`